apiserver/pkg/common/schema.go

```diff
@@ -97,6 +97,11 @@ var (
 			Version:  v1alpha1.GroupVersion.Version,
 			Resource: "llms",
 		},
+		"embedder": {
```

Comment:
pls update code like `schema.GroupVersionResource{Group: v1alpha1.GroupVersion.Group, Version: v1alpha1.GroupVersion.Version, Resource: "embedders"}` to use this helper function